### ISSUE TITLE
enable new lines in argparse text fields

### ISSUE THREAD

**Issue by oinbar** [wooey/Wooey#137]
\n characters are not rendered by browser. It would be nice to display multiline descriptions like in the in command line. 
I am already using formatter_class=RawTextHelpFormatter in the argparse constructor.

**Comment by Chris7** [comment]
I agree. Since it is rendered to HTML, we need to put in a <br> for these cases to emulate the CLI better. I'm currently away for the next few days, but if you want to address this I'd be able to go over the PR. I would add in a new template tag for it.
